addons/stock/models/res_config_settings.py

```diff
@@ -46,7 +46,6 @@ class ResConfigSettings(models.TransientModel):
     annual_inventory_month = fields.Selection(related='company_id.annual_inventory_month', readonly=False)
     annual_inventory_day = fields.Integer(related='company_id.annual_inventory_day', readonly=False)
     group_stock_reception_report = fields.Boolean("Reception Report", implied_group='stock.group_reception_report')
-    group_stock_auto_reception_report = fields.Boolean("Show Reception Report at Validation", implied_group='stock.group_auto_reception_report')
```

Comment:
Don't forget to remove `group_auto_reception_report` from the security file data.

Reply:
I did forget :face_with_head_bandage: Fixed it.